examples/js/loaders/VRMLLoader.js

```diff
@@ -106,8 +106,11 @@ THREE.VRMLLoader = ( function () {
 
 				// from http://gun.teipir.gr/VRML-amgem/spec/part1/concepts.html#SyntaxBasics
 
-				var RouteIdentifier = createToken( { name: 'RouteIdentifier', pattern: /[^\x30-\x39\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d][^\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d]*[\.][^\x30-\x39\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d][^\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d]*/ } );
-				var Identifier = createToken( { name: 'Identifier', pattern: /[^\x30-\x39\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d][^\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d]*/, longer_alt: RouteIdentifier } );
+				// Converted to not use complement sets for performance reasons. See https://sap.github.io/chevrotain/docs/guide/resolving_lexer_errors.html#COMPLEMENT
+				// Original regex: /[^\x30-\x39\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d][^\0-\x20\x22\x27\x23\x2b\x2c\x2d\x2e\x5b\x5d\x5c\x7b\x7d]*
+
+				var RouteIdentifier = createToken( { name: 'RouteIdentifier', pattern: /[\u0021\u0024-\u0026\u0028-\u002a\u002f\u003a-\u005a\u005e-\u007a\u007c\u007e-\uffff][\u0021\u0024-\u0026\u0028-\u002a\u002f-\u005a\u005e-\u007a\u007c\u007e-\uffff]*[\.][\u0021\u0024-\u0026\u0028-\u002a\u002f\u003a-\u005a\u005e-\u007a\u007c\u007e-\uffff][\u0021\u0024-\u0026\u0028-\u002a\u002f-\u005a\u005e-\u007a\u007c\u007e-\uffff]*/ } );
```

Comment:
The previous regex uses the original Unicode hex character codes from the VRML standard. This makes it much easier to read the regex and synchronize it with the standardization document. 

Why have you switched to a different representation (like `\u0021`)?

> // Converted to not use complement sets for performance reasons.

Can you please explain how you have converted the existing regex?